vars/distroVersion.groovy

```diff
@@ -13,17 +13,19 @@ String call() {
 
 String call(String distro) {
     String branch = 'master'
-    if (env.BRANCH_NAME.endsWith('2.2')) {
-        branch = 'release/2.2'
+    if (env.BRANCH_NAME =~ '/[0-9]+\.[0-9]+') {
```

Comment:
I've removed the requirement that the branch name has to end in the version number as presumably we want to support branch names like `weekly-2.4-testing`.

Reply:
FWIW, I name my cherry pick branches like
`dbohning/daos-xxxx-2.2`

Reply:
@daltonbohning For that particular case, the result would probably be correct.  I was thinking more along the lines of somebody creating a PR branch with a name like `someuser/test-of-foobar-3.2` and there being no key in the array matching that.